GoogleCloudPlatform/datacatalog-connectors-bi
# Ingest App assets metadata

**- What I did**
Added a scrape > prepare > ingest workflow for the Qlik Sense Apps metadata. This PR is part of the effort to deliver https://github.com/GoogleCloudPlatform/datacatalog-connectors-bi/issues/34.

**- How I did it**
Added new code to existing classes and unit tests to cover all the new stuff.

**- How to verify it**
Run the unit tests and the connector, then make sure App-related Entries, Tag Template, and Tags were properly created.

**- Description for the changelog**
Added the scrape > prepare > ingest workflow for Qlik Sense Apps metadata.
